### ISSUE TITLE
uuv $pid in Reader.pm

### ISSUE THREAD

**Issue by yoheimuta** [yoheimuta/Linux-GetPidstat#8]
```
Use of uninitialized value $pid in scalar chomp at /path/to/local/lib/perl5/Linux/GetPidstat/Reader.pm line 22.
Use of uninitialized value $pid in pattern match (m//) at /path/to/local/lib/perl5/Linux/GetPidstat/Reader.pm line 88.
Use of uninitialized value $pid in concatenation (.) or string at /path/to/local/lib/perl5/Linux/GetPidstat/Reader.pm line 89.
invalid pid:  at /path/to/local/lib/perl5/Linux/GetPidstat.pm line 48.
Error open (<) on '/var/tmp/make_ad_mute_index': No such file or directory at /path/to/local/lib/perl5/Linux/GetPidstat/Reader.pm line 22.
```

**Comment by yoheimuta** [comment]
I do not mind suppressing it within the range that I use

**Comment by yoheimuta** [comment]
fixed: #9
